source/client/benchmark_client_impl.cc

```diff
@@ -200,9 +157,12 @@ void BenchmarkClientHttpImpl::onPoolFailure(Envoy::Http::ConnectionPool::PoolFai
   case Envoy::Http::ConnectionPool::PoolFailureReason::Overflow:
     benchmark_client_stats_.pool_overflow_.inc();
     break;
-  case Envoy::Http::ConnectionPool::PoolFailureReason::ConnectionFailure:
+  case Envoy::Http::ConnectionPool::PoolFailureReason::LocalConnectionFailure:
+  case Envoy::Http::ConnectionPool::PoolFailureReason::RemoteConnectionFailure:
     benchmark_client_stats_.pool_connection_failure_.inc();
     break;
+  case Envoy::Http::ConnectionPool::PoolFailureReason::Timeout:
```

Comment:
I will add a TODO to track these, as well as add separate counters for the combined connection failure counter above.